pkg/cmd/step/create/step_create_task_integration_test.go

```diff
@@ -0,0 +1,149 @@
+// +build integration
+
+package create
+
+import (
+	"fmt"
+	"github.com/jenkins-x/jx/pkg/cmd/opts"
+	"github.com/jenkins-x/jx/pkg/cmd/step/syntax"
+	"github.com/jenkins-x/jx/pkg/jenkinsfile"
+	"github.com/jenkins-x/jx/pkg/log"
+	"github.com/jenkins-x/jx/pkg/util"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func TestStepCreateTask(t *testing.T) {
```

Comment:
@abayer tried to add an integration test which runs _Run()_ in its simplest form in direct execution as well as in meta pipeline execution mode. Hopefully, this helps to catch errors earlier.